src/iosrv/srv.c

```diff
@@ -91,6 +91,8 @@
 unsigned int	dss_tgt_offload_xs_nr;
 /** Number of target (XS set) per server */
 unsigned int	dss_tgt_nr;
+/** Number of system XS need ctx progress */
```

Comment:
the comment seems not very clear. and global variable is 0 by default already.

Reply:
yes, if there is no spare cores available, it will keep it to zero.